app/views/clans/ClansView.coffee

```diff
@@ -13,32 +13,30 @@ SubscribeModal = require 'views/core/SubscribeModal'
 module.exports = class ClansView extends RootView
   id: 'clans-view'
   template: template
+  publicClansArray: []
+  myClansArray: []
```

Comment:
As in `AnalyticsView` PR, one should never set mutable (and expected to be mutated) arrays or objects in class properties. They should be set in initialize to ensure each instance has their own.